cmd/ipfs-cluster-ctl/formatters.go

```diff
@@ -167,7 +167,12 @@ func textFormatPrintVersion(obj *api.Version) {
 }
 
 func textFormatPrintPin(obj *api.Pin) {
-	fmt.Printf("%s | %s | %s | ", obj.Cid, obj.Name, strings.ToUpper(obj.Type.String()))
+	t := strings.ToUpper(obj.Type.String())
+	if obj.Mode == api.PinModeDirect {
+		t = t + "-DIRECT"
+	}
```

Comment:
We signal that a pin is in `direct` mode by showing `PIN-DIRECT`. Recursive pins will show just the cluster `pin` type as before.